aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1703,15 +1714,16 @@ static Tensor& _linalg_norm_vector_out(Tensor& result, const Tensor& self, optio
       // size is degenerate (like size(0), size(0, N), etc)
       case_was_overridden = true;
       self_ = self_.abs();
```

Comment:
I suspect this is also a perf penalty compared to torch.norm with ord==INFINITY? What are the extreme propagation differences here?